File: Chapter 06/6.4.3 The Modern Architect's AI Rule/Python/models/shipping_request.py
from pydantic import BaseModel, Field

class ShippingRequest(BaseModel):
    product_id: str = Field(
        ..., 
        description="The unique ID of the physical product. Do NOT send digital product IDs (like MP3s or eBooks)."
    )
    zip_code: str = Field(
        ..., 
        description="The destination zip code. Must be exactly 5 digits."
    )

File: Chapter 06/JS/section_6_4_3_the_modern_architect_ai_rule/models/shippingRequest.js
/**
 * THE DTO (The AI's Input Form)
 * * @swagger
 * components:
 * schemas:
 * ShippingRequest:
 * type: object
 * required:
 * - productId
 * - zipCode
 * properties:
 * productId:
 * type: string
 * description: The unique ID of the physical product. Do NOT send digital product IDs (like MP3s or eBooks).
 * zipCode:
 * type: string
 * description: The destination zip code. Must be exactly 5 digits.
 */
module.exports = {}; // Purely structural for Swagger parsing